src/plugins/Dashboard/ActionBrowseTagline.js

```diff
@@ -9,7 +13,10 @@ module.exports = (props) => {
            type="file" 
            name="files[]" 
            multiple="true"
-           onchange=${props.handleInputChange} />`
+           onchange=${props.handleInputChange} 
+           ref=${(input) => {
+             inputEl = input
+           }} />`
```

Comment:
this `<input>` element was made a separate element so that we could store a reference to it in the `input` variable, but that is not necessary when we have `ref`. it could be inlined in the `${input}` line below.